File: python/e10.py
"""
the sum of all primes up to two million.
"""
from primegen import primes_by_sieve

def Euler_10(n = 2e+6):
    #verified(2e+6)
    return sum(primes_by_sieve(n))

def test_Euler_10():
    assert Euler_10() == 142913828922

if __name__ == "__main__":
    print(Euler_10())

File: rust/e10.rs
/*
the sum of all primes up to two million.
*/
mod primegen;

fn euler_10() -> isize {
    let n = 2_000_000;
    primegen::primes_by_sieve(n).iter().sum()
}

fn main() {
    println!("{}", euler_10());
}